packages/plugin/src/generators/executor/executor.ts

```diff
@@ -122,6 +123,18 @@ async function updateExecutorJson(host: Tree, options: NormalizedSchema) {
       options.skipLintChecks
     );
   }
+  // add dependencies
+  updateJson<PackageJson>(
+    host,
+    joinPathFragments(options.projectRoot, 'package.json'),
+    (json) => {
+      json.dependencies = {
+        ...json.dependencies,
+        '@nx/devkit': nxVersion,
```

Comment:
Can we swap the order of these? Or honestly, it may be better to do something like:

```
json.dependencies ??= {};
json.dependencies['@nx/devkit'] ??= readPackageVersionInRootPackageJson(tree, '@nx/devkit')
```

Reply:
I have swapped the order (which might make sense if the user added `@nx/devkit`).

As for the second, I think users will anyway want to adapt their range, so I'd rather not assume anything. Using a fixed version is straightforward and something most people have in their root `package.json` anyway.